wiki/Article_styling_criteria/Formatting/de.md

```diff
@@ -0,0 +1,782 @@
+# Formatierung
+
+*Für die Schriftstandards, siehe: [Artikelgestaltungskriterien/Schrift](../Writing)*
+
+*Hinweis: Dieser Artikel benutzt [RFC 2119](https://tools.ietf.org/html/rfc2119), um die Anforderungen zu beschreiben.*
```

Comment:
this is not applicable if you are not using exact English words mentioned in the document -- same goes for `Writing` and `Article styling criteria` itself

Reply:
on the other hand, I can remove this myself to avoid hindering the PR..